### ISSUE TITLE
Warning: C:\Program Files\Amazon\AWSSAMCLI\runtime\lib\site-packages\chevron\renderer.py:73: SyntaxWarning: "is" with a literal. Did you mean "=="? if scope is 0:

### ISSUE THREAD

**Issue by robertsosinski** [aws/aws-sam-cli#2381]
### Description:

When running sam local start-api I get the following warning:

```
C:\Program Files\Amazon\AWSSAMCLI\runtime\lib\site-packages\chevron\renderer.py:73: SyntaxWarning: "is" with a literal. Did you mean "=="? if scope is 0:
```

### Steps to reproduce:
I am using HttpApi with ```start-api``` with sam version ```1.10.0``` and python version ```3.8.3```

### Observed result:

Everything runs fine, but I get this error.  I fixed it by updating the following code on my machine at ```C:\Program Files\Amazon\AWSSAMCLI\runtime\lib\site-packages\chevron\renderer.py:73```

#### Old Code
```
 if scope is 0:
                return 0
```

#### New Code
```
 if scope == 0:
                return 0
```

Looks like Python is giving a warning due to using ```is``` instead of ```==``` for comparing a number.

### Expected result:

Work without giving the error.

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: Windows 10
2. `sam --version`: 1.10.0
3. `python --version`: 3.8.3

**Comment by aahung** [comment]
This is [chevron](https://github.com/noahmorrison/chevron)'s code which is not under our control. They did this intentionally:

https://github.com/noahmorrison/chevron/blob/a0c11f66c6443ca6387c609b90d014653cd290bd/chevron/renderer.py#L75-L78

It is worth to investigate how to disable this warning though.

**Comment by mndeveci** [comment]
Resolving since it has been released with [v1.11.0](https://github.com/aws/aws-sam-cli/releases/tag/v1.11.0)
